### PR TITLE
fix: prefix 'sa-read' rolebinding with 'appstudio-'

### PR DESCRIPTION
**Pull request by xcoulon**
This particular rolebinding applies to `system:serviceaccounts:${MEMBER_OPERATOR_NAMESPACE`, so it doesn't need to contain any `$USERNAME`, but instead, it should be prefixed with `appstudio-`, as all sibling rolebindings in this template.

Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

see https://github.com/codeready-toolchain/toolchain-e2e/pull/519

### CODE REVIEW

File: deploy/templates/nstemplatetiers/appstudio/ns_appstudio.yaml [hunk 1]
```diff
@@ -131,7 +131,7 @@ objects:
 - apiVersion: rbac.authorization.k8s.io/v1
   kind: RoleBinding
   metadata:
-    name: ${USERNAME}-sa-read
+    name: appstudio-sa-read
```
Comment (alexeykazakov) [inline deploy/templates/nstemplatetiers/appstudio/ns_appstudio.yaml:134]:
```suggestion
    name: member-operator-sa-read
```

**Review by alexeykazakov — COMMENTED** [review]
I would rather call it `member-operator-sa-read` since this one binds all SAs in the member-operator namespace to the `toolchain-sa-read` role.

File: deploy/templates/nstemplatetiers/appstudio/ns_appstudio.yaml (hunk 1 above)
Comment (xcoulon) [reply deploy/templates/nstemplatetiers/appstudio/ns_appstudio.yaml:134]: I thought we would use `appstudio-` for all role bindings related to AppStudio, but I don't have any strong opinion on the names or prefixes.

Comment (xcoulon) [reply deploy/templates/nstemplatetiers/appstudio/ns_appstudio.yaml:134]: ok, got it. As I said, I'm fine with both options ;)

**Review by alexeykazakov — APPROVED** [review]

**Review by MatousJobanek — APPROVED** [review]

**Pull request merged**